lesson3/task4.js

```diff
@@ -0,0 +1,42 @@
+function tW() {
```

Comment:
Классы принято именовать с большой буквы:
```suggestion
function TicketWindow() {
```